features/data_converter/empty/feature.go

```diff
@@ -0,0 +1,55 @@
+package empty
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	historyProto "go.temporal.io/api/history/v1"
+	"go.temporal.io/sdk-features/harness/go/harness"
+	"go.temporal.io/sdk/client"
+	"go.temporal.io/sdk/workflow"
+)
+
+var Feature = harness.Feature{
+	Workflows:   Workflow,
+	CheckResult: CheckResult,
+}
+
+// run a workflow that returns nothing
+func Workflow(ctx workflow.Context) error {
+	return nil
+}
+
+func CheckResult(ctx context.Context, runner *harness.Runner, run client.WorkflowRun) error {
+	// verify client result is null
+	var result interface{}
+	if err := run.Get(ctx, &result); err != nil {
+		return err
+	}
+	if result != nil {
+		return fmt.Errorf("invalid result: %v", result)
+	}
+	history := runner.Client.GetWorkflowHistory(ctx, run.GetID(), "", false, 0)
```

Comment:
We could probably abstract this even further and add a method to get the completion event for a workflow (applies to all lang harnesses)

Reply:
I still am not convinced we need to check payloads. IMO, we only need to check end user behavior. If it comes back as nil, that's fine. (but I understand others disagree w/ me here)

Reply:
I still am not convinced we need to check payloads. IMO, we only need to check end user behavior. If it comes back as nil, that's fine. (but I understand others disagree w/ me here)